package-lock.json

```diff
@@ -208,8 +208,7 @@
 			}
 		},
 		"@automattic/calypso-color-schemes": {
-			"version": "file:packages/calypso-color-schemes",
-			"dev": true
+			"version": "file:packages/calypso-color-schemes"
```

Comment:
It's weird we don't require anything for this package, `@automattic/color-studio` and `@automattic/calypso-build` seem to be used by it.